project_event/models/res_partner.py

```diff
@@ -0,0 +1,16 @@
+# © 2018 Savoir-faire Linux
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import fields, models
+
+
+class ResPartner(models.Model):
+    _name = 'res.partner'
+    _inherit = ['res.partner']
+
+    client_type = fields.Many2one(
+        'res.partner.category',
+        string='Client Type',
+        related='client_type.name',
```

Comment:
the new added field and the related field should have the same type

Reply:
@rimbendhaou Changed